mkdocs_awesome_pages_plugin/navigation.py

```diff
@@ -95,13 +95,24 @@ def _order(self, items: List[NavigationItem], meta: Meta):
         if order is None and sort_type is None and order_by is None:
             return
 
+        ignore_case = meta.ignore_case or self.options.ignore_case
+
         if order_by == Meta.ORDER_BY_TITLE:
-            key = lambda i: self._get_item_title(i)
+            if ignore_case and sort_type != Meta.SORT_NATURAL:
+                key = lambda i: self._get_item_title(i).casefold()
+            else:
+                key = lambda i: self._get_item_title(i)
         else:
-            key = lambda i: basename(self._get_item_path(i))
+            if ignore_case and sort_type != Meta.SORT_NATURAL:
+                key = lambda i: basename(self._get_item_path(i)).casefold()
```

Comment:
Would it be possible to apply the `casefold` at a later stage to avoid the code duplication? Further down where the natural sort is handled.